guides/chalet_time_policy.md

```diff
@@ -1,51 +1,63 @@
- # Chalet time policy
-This document describes where and how people should spend time when not on client work -  known as chalet time. The purpose is to make sure people use this time to build their skills, grow Made Tech’s business and contribute to communities of practice.  
+# Chalet time policy
+
+This document describes where and how people should spend time when not on client work -  known as chalet time. The purpose is to make sure people use this time to build their skills, grow Made Tech’s business and contribute to communities of practice.
+
 ## What is chalet time?
-Chalet time is when people are not on client work. 
+
+Chalet time is when people are not on client work.
 For example, there is a two week gap for someone finishing the software engineering academy and going onto a client project. Another example, a Lead Designer having three days between their current client project and the next.
 
-Occasional chalet time is part of a business like ours, as client work isn’t always going to perfectly match our skills and who is available to work in different teams. 
-However, used the right way, chalet time can be valuable to grow skills, our business and contribute to communities of practice.  
+Occasional chalet time is part of a business like ours, as client work isn’t always going to perfectly match our skills and who is available to work in different teams.
+However, used the right way, chalet time can be valuable to grow skills, our business and contribute to communities of practice.
 
 ## Responsibility for using chalet time usefully
-People with chalet time and their line managers will be responsible to ensure this time is used usefully. 
+
```

Comment:
What is the purpose of the new lines added to this file? When previewed in markdown this looks to have no effect

Reply:
Yeah, there is no visual difference. The reason for the change is to pass [markdown](https://github.com/markdownlint/markdownlint) lint rule [MD022](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines).

There are several files within the repo breaking this rule (and others that are identified by markdown lint) I figured it would be good to make a small improvement when editing the file, HTH 😄.